sdk/core/azure-core/azure/core/_pipeline_client_async.py

```diff
@@ -97,6 +96,7 @@ async def close(self):
         await self._pipeline.__aexit__()
 
     def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
+        from .pipeline import AsyncPipeline
```

Comment:
Same, I feel this one don't need to be put here. Also, this PR claims to be about requests, so sounds strange to have anything in AIO :)